{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -16,3 +16,19 @@ universal = 1
 
 [flake8]
 exclude = docs
+
+[aliases]
+{%- if cookiecutter.use_pytest == 'y' %}
+test = pytest
+{%- endif %}
+# Define setup.py command aliases here
+{%- if cookiecutter.use_pytest == 'y' %}
+
+[pytest]
+# Set permanent options for the "pytest" setup command here
```

Comment:
IMO that pytest warning is a little too aggressive.  A better approach would be to raise the warning only when the section actually contains a known pytest configuration directive.